sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/redis/passwordless/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Spring Cloud Azure's auto-configuration for Azure Redis passwordless connections.
+ */
+package com.azure.spring.cloud.autoconfigure.redis.passwordless;
```

Comment:
I'm not sure whether we need this file here, seems like there's no public API in this package?

Reply:
If it can be deleted, I will remove this file.